src/lib/relay/QueryRenderers.tsx

```diff
@@ -121,6 +121,7 @@ export const BidFlowRenderer: React.SFC<BidderFlowRendererProps> = ({ render, ar
           }
         }
       `}
+      cacheConfig={{ force: true }}
```

Comment:
Can we add this to the `RegistrationFlowRenderer` above as well? It's not likely to cause an issue but is another area that shouldn't be cached.

Reply:
Sure.